draft-ietf-privacypass-architecture.md

```diff
@@ -346,23 +346,30 @@ See {{attester}} for more details.
 
 ### Attester Role {#attester}
 
-Attestation is an important part of the issuance protocol. Attestation is the
-process by which an Attester bears witness to, confirms, or authenticates a
-Client so as to verify a property about the Client that is required for
-Issuance. Examples of attestation properties include, though are not limited
-to:
-
-- Capable of solving a CAPTCHA. Clients that solve CAPTCHA challenges can be
-  attested to have this capability for the purpose of being ruled out as a bot
-  or otherwise automated Client.
-- Client state. Clients can be associated with state and the attester can
-  attest to this state. Examples of state include the number of issuance
-  protocol invocations, the Client's geographic region, and whether the
-  client has a valid application-layer account.
-- Trusted device. Some Clients run on trusted hardware that are capable of
-  producing device-level attestation statements.
-
-Each of these attestation types has different security properties. For example,
+Attestation is an important part of the issuance protocol. In Privacy Pass,
+attestation is the process by which an Attester bears witness to, confirms,
+or authenticates a Client so as to verify a property about the Client that
+is required for Issuance. {{?RFC9334}} describes an architecture for attestation
+procedures. Using that architecture as a conceptual basis, Clients are
+attesters that produce attestation evidence, and Attesters are verififiers that
```

Comment:
I wonder if our Attester also shares some of the RATS Endorser role in this case?

I also think we should disambiguate with adding RATS in the names
```suggestion
RATS attesters that produce attestation evidence, and Attesters are RATS verififiers that
```

Reply:
I think that for the current PAT deployment the attester is also an endorser, but it's not clear to me if that's generally true for all types of attestation procedures.